.github/workflows/index-resources.yml

```diff
@@ -36,4 +68,4 @@ jobs:
             --resourceTypes dataset --collections core staging
       - name: Upload the new index, overwriting the existing index
         run: |
-          aws s3 cp resources.json.gz s3://nextstrain-inventories/resources.json.gz
+          aws s3 cp resources.json.gz s3://nextstrain-inventories/"$APP_NAME"/resources.json.gz
```

Comment:
You'll need to update the IAM users ([e.g. this one for production](https://github.com/nextstrain/nextstrain.org/blob/master/aws/iam/policy/NextstrainDotOrgServerInstance.tftpl.json#L49-L57)) to allow the server to read these files

Reply:
Ah, thank you! I was wondering what other permissions might need to be updated!

Reply:
Updated in https://github.com/nextstrain/nextstrain.org/pull/830/commits/00ab2c9c04ae99f794dc0be1953d1c7ef58754b7, although I have not deployed the changes via Terraform yet.

Reply:
Applied the Terraform changes to the testing environment. 
Now, I can locally start the server with the testing IAM users credentials, pointing to the new resource.json.gz

```shell
RESOURCE_INDEX="s3://nextstrain-inventories/nextstrain-canary/resources.json.gz" \
    envdir ~/Repos/env.d/nextstrain.org-testing/ \
    npm run server
```